src/storage/database.rs

```diff
@@ -57,7 +70,7 @@ impl DatabaseBackend {
         // https://www.postgresql.org/message-id/162867790712200946i7ba8eb92v908ac595c0c35aee%40mail.gmail.com
         let max_size = max_size.min(std::i32::MAX as usize) as i32;
 
-        let rows = if let Some(r) = range {
+        let (path, mime, date_updated, compression, content, is_too_big) = if let Some(r) = range {
```

Comment:
Might be worth declaring a local struct and using `query_as!` here?

Reply:
I did it for now, not 100% certain which looks better, but I lean towards the version _with_ `.query_as!`